__main__.py

```
@@ -0,0 +1,600 @@
+        # Check for base64 patterns
+        b64_pattern = r'[A-Za-z0-9+/]{20,}={0,2}'
+        b64_matches = re.findall(b64_pattern, result_str)
+        if b64_matches:
+            print("\n[*] Found potential base64 strings:")
+            for b64 in b64_matches[:5]:  # Show first 5
+                try:
+                    import base64
+                    decoded = base64.b64decode(b64).decode('utf-8')
+                    if 'flag' in decoded.lower() or 'ctf' in decoded.lower():
+                        print(f"  📦 {b64[:20]}... -> {decoded}")
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)